closed/custom/Docs.gmk

```diff
@@ -191,21 +191,20 @@ OPENJ9_ONLY_JAVADOC_JAVA_ARGS = -Djava.awt.headless=true -Dextlink.spec.version=
 OPENJ9_ONLY_JAVADOC_CMD = $(JAVA) $(OPENJ9_ONLY_JAVADOC_JAVA_ARGS) $(NEW_JAVADOC)
 
 OPENJDK_MODULE_REFERENCE_TEXT := \
-  These are the OpenJ9 extensions to this module. Refer to <a href="$(JAVASE_BASE_URL)">$(JAVASE_BASE_URL)</a> for the reference implementation documentation.
+  These are the OpenJ9 extensions to this module. Refer to <a href="$(JAVASE_BASE_URL)" target="_blank">$(JAVASE_BASE_URL)</a> for the reference implementation documentation.
 
 MODULE_DESCRIPTION_FIRST_LINE  := <section role="region">
-MODULE_DESCRIPTION_SECOND_LINE := MODULE DESCRIPTION
-MODULE_DESCRIPTION_MARKER_TEXT := javadoc_marker_text
-MODULE_DESCRIPTION_LAST_LINE   := </section>
+MODULE_DESCRIPTION_LAST_LINE   := <!-- ============ PACKAGES SUMMARY =========== -->
 MODULE_DESCRIPTION_REPLACE_TEXT := \
   <section class="module-description" id="module.description"> \
     <div class="block"> \
       <dl class="notes"> \
         <dd>$(OPENJDK_MODULE_REFERENCE_TEXT)</dd> \
       </dl> \
     </div> \
-  </section>
-
+  </section> \
+  \$(NEWLINE)<!-- ============ PACKAGES SUMMARY =========== -->
+ 
```

Comment:
Please remove trailing whitespace.

Reply:
Removed the space in the line consisting of a single space (line 207).